server/src/serializers/ReviewsSerializer.js

```diff
@@ -0,0 +1,24 @@
+import UserSerializer from "./UserSerializer.js"
+
+class ReviewsSerializer {
+  static async getSummary(review) {
+    const allowedAttributes = ["title", "body", "rating"]
```

Comment:
In the `HolidaySerilaizer` the `allowedAttributes` array is in the `try` block, but here it is outside of it.
While both work well, we want to keep consistency through the project. Pick one implementation and go with that throughout the project.